danggunMarket/src/main/java/com/example/danggunmarket/common/cache/CacheType.java

```diff
@@ -0,0 +1,18 @@
+package com.example.danggunmarket.common.cache;
+
+import lombok.Getter;
+
+@Getter
+public enum CacheType {
+    Main_PAGE("mainPage", 1, 1000);
```

Comment:
```suggestion
    MAIN_PAGE("mainPage", 1, 1000);
```

보통 열거형 상수는 대문자를 쓰는것이 컨벤션이더라구요.
https://stackoverflow.com/questions/3069743/coding-conventions-naming-enums

Reply:
PAGE는 대문자로 하고 Main에서는 이렇게 하다니 손가락에 힘이 풀렸나봐요